src/neha_Rathi/test/SumOfNumIndex.java

```diff
@@ -0,0 +1,29 @@
+//program1
+package neha_Rathi.test;
+
+public class SumOfNumIndex {
+	
+	int[] calculateIndex(int[] nums, int targetNum) {
+		int[] output = new int[2];
+		for (int index = 0; index < nums.length; index++) {
+			for (int innerIndex = 0; innerIndex < nums.length; innerIndex++)
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files